pinot-controller/src/main/java/org/apache/pinot/controller/helix/SegmentStatusChecker.java

```diff
@@ -61,6 +61,7 @@
 public class SegmentStatusChecker extends ControllerPeriodicTask<SegmentStatusChecker.Context> {
   private static final Logger LOGGER = LoggerFactory.getLogger(SegmentStatusChecker.class);
   private static final int MAX_OFFLINE_SEGMENTS_TO_LOG = 5;
+  private static final String TABLE_CONSUMPTION_PAUSED = "isTablePaused";
```

Comment:
Can we use `isTableConsumptionPaused` to make it easy to read?

Reply:
Actually, this is the key that is set is the table idealstate when a table is paused. So will have to use it to for lookup.